tpm/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+// Copyright © 2022, Microsoft Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+pub mod ioctl;
```

Comment:
Please don't call it `ioctl` if is not about the syscall.

Reply:
I moved all the structs to lib.rs and dropped ioctl.rs. lib.rs seemed to be a better place for these structs and related methods.